src/python/bot/tasks/corpus_pruning_task.py

```diff
@@ -630,6 +639,33 @@ def do_corpus_pruning(context, last_execution_failed, revision):
       fuzzer_binary_name=fuzzer_binary_name,
       revision=environment.get_value('APP_REVISION'))
 
+  sources = ','.join([
+      fuzzer.fuzz_target.project_qualified_name()
+      for fuzzer in context.cross_pollinate_fuzzers
+  ])
+
+  # TODO(mpherman): Add way to get tags string when tagged corpora are done.
+  tags = ''
+
+  if pruner_stats and pollinator_stats:
```

Comment:
It might be a bit cleaner to factor the stats storage into a separate helper function, which will also let you simplify what you're mocking during testing (e.g. you won't need to understand that we're using the API client and mock out the relevant functions there, but can instead just mock out the new helper).